content/applications/finance/accounting/payables/supplier_bills/invoice_digitization.rst

```diff
@@ -16,14 +16,17 @@ created and populated based on the scanned invoices.
      <https://www.odoo.com/slides/slide/digitize-bills-with-ocr-1712>`_.
 
 .. note::
-   - The more documents you scan, the better the system identifies the correct data.
+   It is essential to review and correct (if needed) the information uploaded during digitization.
+   Then, you have to post the document by clicking on :guilabel:`Confirm`. In this manner, the AI
+   learns, and the system identifies the correct data in the future.
 
 Configuration
 =============
 
 In :menuselection:`Accounting --> Configuration --> Settings --> Digitalization`, check the box
 :guilabel:`Document Digitalization` and choose whether :guilabel:`Vendor Bills` and
-:guilabel:`Customer Invoices` should be processed automatically or manually.
+:guilabel:`Customer Invoices` (including Customer Credit Notes) should be processed automatically or
+manually.
 
 The :guilabel:`Single Invoice Line Per Tax` option can also be selected. It enables to get only one
```

Comment:
```suggestion
The :guilabel:`OCR Single Invoice Line Per Tax` option can also be selected. It enables to get only one
```
That's how it's called on 15.0 - could you check that everything else applies to 15.0? This doc was written based on an intermediate release it seems

Reply:
not in V15.3...